src/lib/consts.js

```diff
@@ -6,18 +6,18 @@ const { KEY_VALUE_STORE_KEYS } = require('apify-shared/consts');
 // the name and description should be taken from package.json files in those dirs so that we edit it in a single location!
 // BTW the templates should be clustered by languages, now we have only JS but we'll add other languages later.
 exports.ACTS_TEMPLATES = {
```

Comment:
@mtrunkat @jancurn Can you check renaming templates + new descriptions. Any suggestions for improvements?